macros/AddonSidebar.ejs

```diff
@@ -1,4 +1,4 @@
-<%
+If<%
```

Comment:
Not sure what this `If` is for.

Reply:
Story, must've been a stray copy paste, removed in latest commit